Update index.md (#71680)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

Custom branding allows you to replace the Grafana brand and logo with your own corporate brand and logo.

{{% admonition type="note" %}}

> Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud). Custom Branding is available in Grafana Cloud and Enterprise. For Cloud Advanced/Enterprise customers, please provide your desired custom elements and corresponding logos to our Support team. We will assist in hosting your images and updating your Custom Branding.

Reviewer: ```suggestion
Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud). For Cloud Advanced and Enterprise customers, please provide your desired custom elements and corresponding logos to our Support team. We will help you in hosting your images and updating your Custom Branding.
```
